pages/automatedtestingcodeceptjs.md

```diff
@@ -121,4 +125,13 @@ Awesome!
 While CodeceptJS works awesomely for most websites, it presents some problems to work with BeLL-Apps.   
 Specifically, when trying to load the homepage, the Selenium Server throws an error stating that jQuery is not loaded, and then it goes on throwing many more errors stating `ReferenceError: $ is not defined`.  
 Most likely, this depends on the BeLL-Apps codebase and it is not related to CodeceptJS, so it would probably be worth investigating further to find out the root cause of the error.
+Another problem is that testing takes a long time in running with Bell-Apps. This leads to some test cases failed because of loading time problem.
+
+## Common bugs
+
+If you experience some errors, this may be because of the following reasons:
+1. Selenium, Firefox, Node JS, geckodriver are not updated to the appropriate version.
+2. Bell-Apps take a long time to load.
+3. Selenium, or Firefox, or Node JS, or geckodriver has not running in the system yet.
 
+If you facing any errors, please contact other interns through _gitter chat_, and may specifically ask for **@lmmrssa** or **@duongdo27**.
```

Comment:
If you _face_ any errors, please contact other interns through gitter- specifically ask for **@lmmrssa** or **@duongdo27**.